chpl/chpl-resources/src/main/resources/jobs.xml

```diff
@@ -781,25 +781,6 @@
             <recover>false</recover>
         </job>
 
-        <job>
```

Comment:
I think we need something at the top of this file like `<delete-job>` to make the job and triggers go away without resetting quartz